### PR TITLE
Dev 49484 add the rbac for the minishift case

### PR DESCRIPTION
**Pull request by JeremyTangCD**
Dev 49484 add the rbac for the minishift case

### CODE REVIEW

**Comment by JeremyTangCD** [comment]
@woz5999 
I do not know why the change in https://github.com/logicmonitor/k8s-helm-charts/pull/28 still exists in this PR.
Could you help to check it? Thx

File: argus/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Automated Kubernetes monitoring
 name: argus
-version: 0.3.0
+version: 0.5.0
```
Comment (JeremyTangCD) [inline argus/Chart.yaml]: Pls check if the version change is ok

File: collectorset-controller/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Automated collector management
 name: collectorset-controller
-version: 0.1.0
+version: 0.2.0
```
Comment (JeremyTangCD) [inline collectorset-controller/Chart.yaml]: @woz5999 
Pls check if the version change is ok

File: argus/Chart.yaml (hunk 1 above)
Comment (woz5999) [reply argus/Chart.yaml]: why are we skipping a minor version? Should be 0.4.0

Comment (JeremyTangCD) [reply argus/Chart.yaml]: Actually, we had changed the minor version to 0.4.0 in https://github.com/logicmonitor/k8s-helm-charts/pull/28/files 
I do not know why the master branch still show the previous file

**Comment by woz5999** [comment]
@JeremyTangCD There are merge conflicts that need to resolved

**Pull request closed without merging**